packages/angular/README.md

```diff
@@ -1,28 +1,146 @@
-# JSONForms - More Forms. Less Code
+# JSON Forms - More Forms. Less Code
 
 ### Complex Forms in the blink of an eye
 
-JSONForms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+JSON Forms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
 
-# Angular Package
+## Angular Package
 
-This is the JSONForms Angular package which provides the necessary bindings for Angular. It uses [JSONForms Core](https://www.npmjs.com/package/@jsonforms/core).
+This is the JSON Forms Angular package which provides the necessary bindings for Angular. It uses [JSON Forms Core](https://www.npmjs.com/package/@jsonforms/core).
 
-You can combine the Angular package with any Angular-based renderer set you want, for example the [Material Renderers](https://github.com/eclipsesource/jsonforms/blob/master/packages/angular-material).
+You can combine the Angular package with any Angular-based renderer set you want, for example the [Material Renderers](https://www.npmjs.com/package/@jsonforms/angular-material).
 
-# License
+See the official [documentation](https://jsonforms.io/) and the JSON Forms Angular [seed repository](https://github.com/eclipsesource/jsonforms-angular-seed) for examples on how to integrate JSON Forms with your application.
 
-The JSONForms project is licensed under the MIT License. See the [LICENSE file](https://github.com/eclipsesource/jsonforms/blob/master/LICENSE) for more information.
+Check <https://www.npmjs.com/search?q=%40jsonforms> for all published JSON Forms packages.
 
-# Roadmap
+### Usage
 
-Our current roadmap is available [here](https://github.com/eclipsesource/jsonforms/blob/master/ROADMAP.md).
+Use the `JsonForms` component to render a form for your data.
+
+Mandatory props:
+
+* `data: any` - the data to show
+* `renderers: JsonFormsRendererRegistryEntry[]` - the Angular renderer set to use
+
+Optional props:
+
+* `schema: JsonSchema` - the data schema for the given data. Will be generated when not given.
+* `uischema: UISchemaElement` - the UI schema for the given data schema. Will be generated when not given.
+* `cells: JsonFormsCellRendererRegistryEntry[]` - the Angular cell renderer set to use
+* `config: any` - form-wide options. May contain default ui schema options.
+* `readonly: boolean` - whether all controls shall be readonly.
+* `uischemas: JsonFormsUiSchemaEntry[]` - registry for dynamic ui schema dispatching
+* `validationMode: 'ValidateAndShow' | 'ValidateAndHide' | 'NoValidation'` - the validation mode for the form
+* `ajv: AJV` - custom Ajv instance for the form
+* `refParserOptions: RefParserOptions` - configuration for ref resolving
+* `onChange` - callback which is called on each data change, containing the updated data and the validation result.
```

Comment:
Are these really options in the Angular set? The list of mandatory and optional props needs to be checked against the code.